release.nix

```diff
@@ -0,0 +1,25 @@
+{ witnessUrl ? null
+, buildYarnPackage
+, parallel
+, brotli
+, constGitIgnore
+}:
+
+buildYarnPackage {
+  WITNESS_API_URL = witnessUrl;
+  src = constGitIgnore "validatorcv" ./. [];
+
+  buildInputs = [ parallel brotli ];
+
+  postBuild = ''
+    find build/ -type f \
+      -not -name '*.jpg' \
+      -not -name '*.png' \
+      -not -name '*.webp' \
+      -not -name '*.woff' \
+      -not -name '*.woff2' | parallel brotli
+  '';
+
+  yarnBuildMore = "yarn build";
```

Comment:
maybe you should run `WITNESS_API_URL=your_fantastic_url yarn build` where WITNESS_API_URL will be process.env.WITNESS_API_URL in my super-hero-code?)

Reply:
That environment variable is made available at line 9.